### PR TITLE
Fix the reference assembly for mscorlib to match the surface area of the OS

### PR DESCRIPTION
**Pull request by AlexGhiondea**
This will enable the facade generation across the OSes.

### CODE REVIEW

**Comment by mmitche** [comment]
@AlexGhiondea @mikem8361 is this the same mac error we've been seeing for a while?

**Comment by mikem8361** [comment]
This is a failure in the PAL tests not the semaphore/disk full problem. I think @sergiy-k is looking at this one.

File: src/.nuget/Microsoft.TargetingPack.Private.CoreCLR/Microsoft.TargetingPack.Private.CoreCLR.pkgproj [hunk 1]
```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <Version>1.0.1</Version>
+    <SkipPackageFileCheck>true</SkipPackageFileCheck>
+    <SkipValidatePackage>true</SkipValidatePackage>
+    <PackagePlatforms>x64;x86;arm</PackagePlatforms>
```
Comment (weshaggard) [inline src/.nuget/Microsoft.TargetingPack.Private.CoreCLR/Microsoft.TargetingPack.Private.CoreCLR.pkgproj]: We shouldn't need all these architectures. Just one should do as it is just a reference assembly.

File: src/.nuget/Microsoft.TargetingPack.Private.CoreCLR/Microsoft.TargetingPack.Private.CoreCLR.pkgproj [hunk 2]
```diff
@@ -0,0 +1,20 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <PropertyGroup>
+    <Version>1.0.1</Version>
```
Comment (weshaggard) [inline src/.nuget/Microsoft.TargetingPack.Private.CoreCLR/Microsoft.TargetingPack.Private.CoreCLR.pkgproj]: Assuming you didn't already we should bump this version with the move.

Comment (AlexGhiondea) [inline src/.nuget/Microsoft.TargetingPack.Private.CoreCLR/Microsoft.TargetingPack.Private.CoreCLR.pkgproj]: I bumped this already.

File: src/.nuget/Microsoft.TargetingPack.Private.CoreCLR/Microsoft.TargetingPack.Private.CoreCLR.pkgproj (hunk 1 above)
Comment (AlexGhiondea) [inline src/.nuget/Microsoft.TargetingPack.Private.CoreCLR/Microsoft.TargetingPack.Private.CoreCLR.pkgproj]: Fixed.

**Pull request merged**